### PR TITLE
Fix platform external links

### PR DESCRIPTION
**Pull request by sean-binary**
Changes:

-   ...

## Type of change

-   [x] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/home/platform-carousel/_utils.tsx [hunk 1]
```diff
@@ -95,7 +92,7 @@ export const platform_details_cr: TPlatformDetails[] = [
         description: <Localize translate_text="The CFD trading platform to fit your style." />,
         learn_more_link: '/derivx/',
         download_links: [
-            { type: 'browser', url: derivx_app_url },
+            { type: 'browser', link_type: 'deriv_app' },
```
Comment (njazuli-deriv) [inline src/pages/home/platform-carousel/_utils.tsx:95]: Just to clarify, we don't have `derivx_app` type for this one?

Comment (sean-binary) [reply src/pages/home/platform-carousel/_utils.tsx:95]: We do! Mistaken that for Deriv app. Will change it, thanks for spotting it!

**Review by njazuli-deriv — APPROVED** [review]
LGTM

**Review by njazuli-deriv — APPROVED** [review]
Desktop and mobile view looks good now.

**Pull request merged**